### PR TITLE
Add permission and support for "ftw.lawgiver".

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: ftw/htmlblock/browser/htmlblock.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from zope.publisher.browser import BrowserView
+
+
+class HtmlBlockView(BrowserView):
+    def __call__(self):
+        return self.index()
```
Comment (jone) [inline ftw/htmlblock/browser/htmlblock.py:6]: I think this is default behavior (the call method of BrowserView does the same), therefore superfluous.
Actually you could probably not have a class at all for the view and just use the page template 😉

File: ftw/htmlblock/profiles/default/rolemap.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<rolemap>
+    <permissions>
+        <permission name="ftw.htmlblock: Add HTML block" acquire="False">
+            <role name="Manager"/>
+        </permission>
```
Comment (jone) [inline ftw/htmlblock/profiles/default/rolemap.xml:7]: Let's also add `Site Manager` here by default (thinking about who crafts site roots).

File: ftw/htmlblock/browser/htmlblock.py (hunk 1 above)
Comment (mbaechtold) [inline ftw/htmlblock/browser/htmlblock.py:6]: This has been changed in #2

File: ftw/htmlblock/profiles/default/rolemap.xml (hunk 2 above)
Comment (mbaechtold) [inline ftw/htmlblock/profiles/default/rolemap.xml:7]: Done :+1:

**Comment by jone** [comment]
nice

**Pull request merged**